README.md

```diff
@@ -10,11 +10,15 @@ For a detailed explanation and walkthrough of each scenario, refer to our [Cosmo
     The Wiki Cohere English Embedding Only Scenario contains 768 dimensional embeddings of English Wikipedia articles (without corresponding passage text).
     The embeddings have been generated using Cohere’s multilingual-22-12 model. 
     
-    For simplicity, we use pre-processed version of the dataset hosted at [BigANN](https://github.com/harsha-simhadri/big-ann-benchmarks/blob/main/benchmark/datasets.py).
     This dataset contains :
     - Base data slices of sizes [100K, 10Million and 35Million].
     - Query vectors and corresponding ground truth neighbor identifiers / distances for 5000 vectors not in the base dataset.
+2. MS-Turing-EmbeddingOnly Scenario :
+   Microsoft Turing-ANNS-1B is a dataset being released by the Microsoft Turing team. It consists of Bing queries encoded by Turing AGI v5 that trains Transformers to capture similarity of intent in web search queries. 

```

Comment:
removed being

Reply:
" that trains Transformers to capture similarity of intent in web search queries."

Isn't this obvious and can be removed?